LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift

```diff
@@ -50,6 +50,23 @@ public struct BasalRateScheduleEditor: View {
         self.save = save
         self.mode = mode
     }
+    
+    public init(
+        viewModel: TherapySettingsViewModel,
+        mode: PresentationMode = .acceptanceFlow
```

Comment:
_Technically_, `mode` is in `TherapySettingsViewModel`.  Consider getting it from there instead. (applies to all of these initializers)